### PR TITLE
LPS-10010

### PR DESCRIPTION
**Pull request by Samireddebbarhi**

### CODE REVIEW

**Comment by pedroamarall** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/MyBlogTests.testcase [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+				locator1 = "BlogsEntry#TAG");
+		}
+	}
+
+	@description = "Can Delete a Blog"
+	@priority = 5
+	test CanDeleteBlog {
+		task ("Given the Blogs Portlet is added to the page And Blog Entry exists ") {
+			MyBlogs.Addblog();
+		}
```
Comment (AkramYoussoufi) [inline portal-web/test/functional/com/liferay/portalweb/tests/MyBlogTests.testcase:52]: here you need to use a JSON

File: portal-web/test/functional/com/liferay/portalweb/tests/MyBlogTests.testcase [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+		task("Then the blog Entry should be edited and displayed") {
+            AssertElementPresent(
+				key_entryTitle = "New title",
+				key_tagName = "New title",
+				locator1 = "BlogsEntry#TAG");
+
+		}
+	}
+
+	@description = "Can Search Blogs"
```
Comment (AkramYoussoufi) [inline portal-web/test/functional/com/liferay/portalweb/tests/MyBlogTests.testcase:93]: make a more descriptive name for what the test gonna do!

File: portal-web/test/functional/com/liferay/portalweb/paths/ConfigurationPanel.path [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+</tr>
+<tr>
+	<td>BUTTON_CONTROL</td>
+	<td>//*[contains(@class, 'applications-menu-header')]//*[contains(text(),'Control Panel')]</td>
+	<td></td>
+</tr>
+</tbody>
+</table>
+</body>
+</html>
```
Comment (AkramYoussoufi) [inline portal-web/test/functional/com/liferay/portalweb/paths/ConfigurationPanel.path:31]: TIP: use contain

**Review by AkramYoussoufi — COMMENTED** [review]
you only need to apply some small changes

**Comment by AkramYoussoufi** [comment]
ci:test:sf

**Comment by Youness-ELMO** [comment]
ci:test

**Pull request closed without merging**